scripts/__tests__/gridFilter-test.js

```diff
@@ -6,24 +6,38 @@ var GridFilter = require('../gridFilter.jsx');
 var TestUtils = React.addons.TestUtils;
 
 describe('GridFilter', function(){
-	var filter; 
-	beforeEach(function(){
-	    filter = TestUtils.renderIntoDocument(<GridFilter />);
-	});
-
-	it('calls change filter when clicked', function(){
-		var mock = jest.genMockFunction(); 
-		filter.props.changeFilter = mock;
-
-		var someEvent = {
-			"target":{
-				"value":"hi"
-			}
-		};
-
-		var input = TestUtils.findRenderedDOMComponentWithTag(filter, 'input');		
-		React.addons.TestUtils.Simulate.change(input, someEvent);
-
-		expect(mock.mock.calls).toEqual([["hi"]]);
-	})
-});
\ No newline at end of file
+    var filter;
```

Comment:
Sorry for the confusing diff but the whitespace was updated based on the `.editorconfig`. Seems like there is a mix of tabs and spaces throughout the project.